algebra/big_operators.lean

```diff
@@ -42,7 +42,7 @@ by simp [finset.prod]
 attribute [to_additive finset.sum_const_zero] prod_const_one
 
 @[simp, to_additive finset.sum_image]
```

Comment:
I guess this and the next lemmas should be proven on `multiset` and then derived from this. Then these `classical.dec_eq` are not necessary anymore.

Reply:
Is this comment meant to be somewhere else? `prod_image` doesn't mention `classical.dec_eq`, and the `decidable_eq` is necessary to state the theorem. In general, what's the policy on lemmas that require decidability for the proof, but not the statement? I thought the policy was that classical logic was fine even when unnecessary, so there are a few lemmas where I just used `classical.dec_eq` in the proof, to save the small bother of a user having to provide the `dec_eq`.

Reply:
Oh sorry, my comment on `classical.dec_eq` was unnecessary.
For me using `classical.dec_eq` is also fine, just a little bit annoying..